### ISSUE TITLE
Comet 2024.01 stalls during search

### ISSUE THREAD

**Issue by cbielow** [UWPR/Comet#62]
When running our test suite, the latest Comet version stalls at 2% of the search and runs indefinitely.
Previous Comet versions ran fine on the same input.

```
comet.exe  ../openms/share/OpenMS/examples/FRACTIONS/BSA1_F1.mzML
```

This happens on both Windows and Linux. Adding some sanity checks (`-D_GLIBCXX_ASSERTIONS -g -fsanitize=address` on Linux does not reveal any immediate problems). 

I will attach the [comet.params.txt](https://github.com/user-attachments/files/16096018/comet.params.txt) here.
The mzML can be downloaded [here](https://github.com/OpenMS/OpenMS/blob/develop/share/OpenMS/examples/FRACTIONS/BSA1_F1.mzML), the FASTA [here](https://github.com/OpenMS/OpenMS/blob/develop/share/OpenMS/examples/TOPPAS/data/BSA_Identification/18Protein_SoCe_Tr_detergents_trace_target_decoy.fasta)

The issue seems unrelated to #60 and #59 (i.e. it does not matter what is used as peptide_mass_tolerance)

**Comment by jke000** [comment]
Thanks Chris. Fix for this is known; it will be committed soon.

**Comment by jke000** [comment]
This is addressed by commit e5cf236.
